src/diffusers/models/unets/unet_3d_blocks.py

```diff
@@ -1574,7 +1574,7 @@ def __init__(
                     num_attention_heads=temporal_num_attention_heads,
                     in_channels=out_channels,
                     num_layers=temporal_transformer_layers_per_block[i],
-                    norm_num_groups=temporal_norm_num_groups,
+                    norm_num_groups=resnet_groups,
```

Comment:
Context for why we need this change: https://github.com/huggingface/diffusers/pull/7707#discussion_r1570043527. It is the correct thing to do here. cc @DN6